doc/interop-test-descriptions.md

```diff
@@ -270,12 +274,12 @@ with desired oauth scope.
 Server features:
 * [UnaryCall][]
 * [Compressable Payload][]
-* SimpeResponse.username
-* SimpleResponse.oauth_scope
+* Echoes authenticated username in SimpeResponse.username
+* Echoes OAuth scope in SimpleResponse.oauth_scope
 
 Procedure:
- 1. Client sets flags default_service_account with GCE service account name and
-    oauth_scope with the oauth scope to use.
+ 1. Client sets --default_service_account with GCE service account email and
```

Comment:
This step should be removed. "Client" makes it seem like this is something the test is supposed to do to itself, which is obviously not the case. But then the test doesn't do anything to satisfy this step.